@here/harp-mapview/lib/text/MapViewState.ts

```diff
@@ -33,7 +33,7 @@ export class MapViewState implements ViewState {
         return this.m_mapView.frameNumber;
     }
     get lookAtDistance(): number {
-        return this.m_mapView.lookAtDistance;
+        return this.m_mapView.targetDistance;
```

Comment:
As mentioned above: Don't rename 'lookAtDistance` for now.
I was also thinking to reverse the logic of the MapViewState, i.e.
MapViewState should be a (private) property of MapView and could contain more stuff that is needed internally only. The MapView getters would then just forward the MapViewState properties.

This would allow us to store more properties and pass them around w/o polluting the public MapView API. lookAtDistance is a good example of a property that should have never been exposed b/c it is for internal use only.